Summit Demo/streamlit_app.py

```diff
@@ -0,0 +1,71 @@
+import streamlit as st
+import numpy as np
+from snowflake.snowpark.context import get_active_session
+from snowflake.cortex import Complete
+import pandas as pd
+st.set_page_config(layout="wide")
+
+def is_local() -> bool:
+    """
+    Check if app is running locally, by checking if the user email is a local one.
+
+    Returns:
+        bool: True if running locally, else (if in SiS) False.
+    """
+    return st.experimental_user.email in {"test@localhost.com", "test@example.com"}
+
+st.title(f"❄️ Streamlit in Snowflake Key Metrics (dummy data) ❄️")
+st.sidebar.image('https://upload.wikimedia.org/wikipedia/commons/f/ff/Snowflake_Logo.svg')
+
+if is_local():
+    conn = st.connection('snowflake')
+    session = conn.session()
+
+else:
+    session = get_active_session()
+
+
+#create some dummy customer data for this demo
+df = pd.DataFrame({
```

Comment:
Would be good to:
1. Add a random seed so that the numbers are consistent between runs
2. Cache this data after generating the first time -- probably not necessary for performance in this case, but good to show it as a good example